x/bank/keeper/grpc_query.go

```diff
@@ -56,6 +56,11 @@ func (k BaseKeeper) AllBalances(ctx context.Context, req *types.QueryAllBalances
 
 	pageRes, err := query.Paginate(accountStore, req.Pagination, func(key, value []byte) error {
 		denom := string(key)
+		if req.IbcResolve {
```

Comment:
Maybe we should add a comment, from which version of IBC, there is a denom metadata registered automatically.